zcash_client_sqlite/src/wallet/init/migrations/shardtree_support.rs

```diff
@@ -184,6 +189,27 @@ impl RusqliteMigration for Migration {
             }
         }
 
+        // Establish the scan queue & wallet history table
+        transaction.execute_batch(
+            "CREATE TABLE scan_queue (
+                block_range_start INTEGER NOT NULL,
+                block_range_end INTEGER NOT NULL,
+                priority INTEGER NOT NULL,
+                CONSTRAINT range_start_uniq UNIQUE (block_range_start),
+                CONSTRAINT range_end_uniq UNIQUE (block_range_end),
+                CONSTRAINT range_bounds_order CHECK (
+                    block_range_start < block_range_end
+                )
+            );",
+        )?;
+
+        if let Some((start, end)) = block_height_extrema(transaction)? {
```

Comment:
Check for a potential off-by-one here if `block_height_extrema` returns an inclusive end height.

Reply:
There was indeed an off-by-one here.